Symfony/CS/Fixer.php

```diff
@@ -286,12 +287,12 @@ public function fixFile(\SplFileInfo $file, array $fixers, $dryRun, $diff, FileC
                 return;
             }
 
-            if (!$dryRun && false === @file_put_contents($file->getRealpath(), $new)) {
+            if (!$dryRun && false === @file_put_contents($file->getRealPath(), $new)) {
```

Comment:
fixing the case of the method name is unrelated to adding integration tests. You should submit it as a separate PR